addons/website/views/snippets/s_image_gallery.xml

```diff
@@ -37,20 +37,20 @@
 </template>
 
 <template id="s_images_wall" name="Images Wall">
-    <section class="s_image_gallery o_spc-small o_masonry pt24 pb24" data-vcss="002" data-columns="3" style="overflow: hidden;">
-        <div class="container">
+    <section class="s_image_gallery o_spc-small o_masonry pt24 pb24 o_disable_snippet_insertion" data-vcss="002" data-columns="3" style="overflow: hidden;">
+        <div class="container o_disable_typing">
```

Comment:
Changes in view in stable version is not recommended. This should have been introduced via JS and removed in master.